server/master/src/main/java/org/apache/accumulo/master/metrics/Metrics2ReplicationMetrics.java

```diff
@@ -58,6 +59,7 @@ protected void snapshot() {
     registry.add(PENDING_FILES, getNumFilesPendingReplication());
     registry.add(NUM_PEERS, getNumConfiguredPeers());
     registry.add(MAX_REPLICATION_THREADS, getMaxReplicationThreads());
+    registry.add(LATENCY, getLatencyInSeconds());
```

Comment:
Also, now that I think of this, "latency" is not the right word to use. Replication latency should be the time in which some record (maybe file, for simplicity) was created to when it was replicated. The "latency" which you have here is not that -- it's the current maximum time from WAL creation to being the replication record being cleaned up. This maximum value includes time for Accumulo to clean up the record as well as the time when a WAL was created before it received any data for a table being replicated -- it's an over-estimation.